EngineLayer/CrosslinkSearch/CrosslinkSearchEngine.cs

```diff
@@ -293,13 +324,16 @@ private void FindCrosslinkedPeptide(Ms2ScanWithSpecificMass scan, List<int> idsO
                         betaMassLowIndex++;
                     }
                 }
+
+                rank++;
             }
         }
+        
 
         /// <summary>
         /// Localizes the crosslink position on the alpha and beta peptides
         /// </summary>
-        private CrosslinkSpectralMatch LocalizeCrosslinkSites(Ms2ScanWithSpecificMass theScan, int alphaIndex, int betaIndex, Crosslinker crosslinker)
+        private CrosslinkSpectralMatch LocalizeCrosslinkSites(Ms2ScanWithSpecificMass theScan, int alphaIndex, int betaIndex, Crosslinker crosslinker, double[] xs, int[] intensityRanks)
```

Comment:
"xs" should have a more descriptive name. Unclear what the variable is used for.

Reply:
Name changed.